app/assets/javascripts/components/buttons/form-button-component.js

```diff
@@ -0,0 +1,47 @@
+ManageIQ.angular.app.component('formButtonComponent', {
```

Comment:
Just a tiny detail.. `formButtonsComponent` .. since this is not a button component, it is a component with a bunch of buttons.